nvm.sh

```diff
@@ -2007,6 +2014,11 @@ nvm_install_source() {
   local TMPDIR
   local VERSION_PATH
 
+  if [ "${NVM_NO_PROGRESS-}" = "1" ]; then
+    PROGRESS_BAR="-sS"
+  else
+    PROGRESS_BAR="--progress-bar"
```

Comment:
where is "PROGRESS_BAR", as defined here, referenced?

Reply:
You mean the definition?

Reply:
no - I mean that lines 2017-2021 seem useless since nothing ever uses `PROGRESS_BAR` in this code path.

Reply:
It can't use it unless it's passed or exported.